### ISSUE TITLE
New version of .travis.yml

### ISSUE THREAD

**Issue by jeffreypullin** [stan-dev/rstantools#50]
Hi, 

I have been using the `mlysy-rstantools_patch` of the repository after being put onto it [here](https://discourse.mc-stan.org/t/problem-running-devtools-install-on-simple-rstan-based-package/7046/5) to build my new **rstan** pacakge [rater](https://github.com/Voltemand/rater). Overall the branch works great but I've had trouble using the included `.travis.yml` file. Some issues it has had are:

```
E: Unable to locate package llvm-5.0-dev
```

```
gcc: error: unrecognized command line option ‘-stdlib=libc++’
```

while building many of the dependencies of **devtools**

```
/usr/bin/ld: /usr/local/clang-5.0.0/bin/../lib/LLVMgold.so: error loading plugin: /usr/local/clang-5.0.0/bin/../lib/LLVMgold.so: cannot open shared object file:
```

while building **Rcpp**

I understand this may be very low priority but I suspect others will have the same issues especially once this branch is merged into master 

Thanks!

Edit: Didn't see #38 before posting this but I will leave it here as it contains more bugs than that issue

**Comment by andrjohns** [comment]
Closing this as we'll be removing the `travis.yml`, following #89
